docs/api-docs/tax/tax-settings.mdx

```diff
@@ -2,6 +2,12 @@
 
 Tax settings allow you to choose how a store handles tax calculation and price display. For example, you can set how a store handles taxes if an enabled tax service is unavailable. You can also choose whether prices include taxes when displayed on orders and invoices. For more info, see [Tax Settings](https://support.bigcommerce.com/s/article/Tax-Overview?language=en_US#tax-settings).
 
+## Estimate taxes and display prices based on tax zone
+
+The Tax Settings API supports cross-border transactions. You can localize storefront prices for shoppers from different regions. For example, you can display tax-inclusive or exclusive prices depending on tax zone. This benefits single-storefront stores that sell to shoppers from different regions. This functionality applies to Stencil and GraphQL-powered storefronts that use basic tax.
```

Comment:
`depending on the tax zone`
or 
`depending on a given tax zone`

Reply:
Thanks, nice catch